miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt

```diff
@@ -49,6 +49,26 @@ class MiniAppFileChooserDefault(var requestCode: Int) : MiniAppFileChooser {
             if (fileChooserParams?.mode == WebChromeClient.FileChooserParams.MODE_OPEN_MULTIPLE) {
                 intent?.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, true)
             }
+            // Uses Intent.EXTRA_MIME_TYPES to pass multiple mime types.
+            fileChooserParams?.acceptTypes?.let { acceptTypes ->
+                if (!acceptTypes.equals("") && acceptTypes.size > 1) {
+                    // Accept all first.
+                    intent?.type = "*/*"
+                    val imageExtensionList = listOf(".jpg", ".jpeg", ".png")
```

Comment:
This is only going to work for `image/` mime types - also this still doesn't cover all image types such as svg, gif, etc. It's possible someone will want to use this for audio, video, or any other type of files as well. So we would need to update this function every time that we need to support another mime-type.

Could you instead use [MimeTypeMap.getMimeTypeFromExtension](https://developer.android.com/reference/android/webkit/MimeTypeMap#getMimeTypeFromExtension(java.lang.String)) to convert the extension to a mime type?

Reply:
@corycaywood updated the code for any type of files now..